src/new/wallet.js

```diff
@@ -230,6 +230,69 @@ class HathorWallet extends EventEmitter {
     }
   }
 
+  getAddressInfo(address, options = {}) {
+    storage.setStore(this.store);
+    const { token = HATHOR_TOKEN_CONFIG.uid } = options;
+
+    // Throws an error if the address does not belong to this wallet
+    if (!this.isAddressMine(address)) {
+      throw new Error('Address does not belong to this wallet.');
+    }
+
+    // Derivation path index
+    const index = this.getAllAddresses().indexOf(address);
+
+    // All transactions for this address
+    const historyTransactions = Object.values(this.getTxHistory());
+
+    // Address information that will be calculated bellow
+    const addressInfo = {
+      total_amount_received: 0,
+      total_amount_sent: 0,
+      total_amount_available: 0,
+      total_amount_locked: 0,
+      token,
+      index
+    };
+
+    // Iterate through transactions
+    historyTransactions.forEach(transaction => {
+      // Voided transactions should be ignored
+      if (transaction.is_voided) {
+        return;
+      };
+
+      // Iterate through outputs
+      transaction.outputs.forEach(output => {
+        const is_address_valid = output.decoded.address === address;
```

Comment:
Not all outputs are have `txout.decoded`. We must handle this case here.

Reply:
Fixed